### DISCUSSION TITLE
How to add XuanFang Features to this system?

### DISCUSSION THREAD

**Discussion by ooemir23** [mathoudebine/turing-smart-screen-python#474] [Ideas]
@mathoudebine 
![mscreen](https://github.com/mathoudebine/turing-smart-screen-python/assets/74644887/543e5f1f-a09a-42f1-b0be-9719a28a05fb)
First of all, thank you for your beautiful and successful work.
I am using XuanFang version display. It has a lot of features in its theme customization. I wonder if you can add these to your system?
For example, I would love to see the frequency of my Graphics Card.

I tried to combine the screenshot with paint =) and sory my bad english.. İm using translate.

**Comment by ooemir23** [comment]
Maybe I can explain it better this way. Because I was able to do this in XuanFang's bad practice.

![mscreen2](https://github.com/mathoudebine/turing-smart-screen-python/assets/74644887/6f2321ee-fd06-4d63-9fa0-0964348feb16)

**Comment by mathoudebine** [comment]
Hello, there is a feature request here #103
I will try to implement this when I have time!
